lib/cc/cli/init.rb

```diff
@@ -64,9 +64,8 @@ def create_default_engine_configs
       end
 
       def available_engine_configs
-        engine_names = existing_cc_config.engines.select do |_, config|
-          config.enabled?
-        end.keys
+        engines = existing_cc_config.engines || {}
```

Comment:
Ah cool. Makes sense! I think returning nil is fine. Coercing it into `{}` might harm us more down the road.